weblab_cg/_chaste_printer.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, symbol_function=None, derivative_function=None):
         self._prefix = ''
 
         # Make sure we can output a call to GetIntracellularAreaStimulus
-        Printer._custom_functions.add('GetIntracellularAreaStimulus')
+        Printer._custom_functions.update({'GetIntracellularAreaStimulus': 'GetIntracellularAreaStimulus'})
```

Comment:
Why not `_custom_functions['GetIntracellularAreaStimulus'] = 'Get...' ?

Reply:
no particular reason :) I'll change it.